src/github.rs

```diff
@@ -146,8 +147,65 @@ impl GitHubApi {
         }
         Ok(result)
     }
+
+    /// Get all teams for the rust-lang org
+    pub(crate) fn teams(&self) -> Result<Vec<GitHubTeam>, Error> {
+        Ok(self
+            .prepare(true, Method::GET, "orgs/rust-lang/teams?per_page=100")?
+            .send()?
+            .error_for_status()?
+            .json()?)
+    }
+
+    /// Get all team members for the team with the given id
+    pub(crate) fn team_members(&self, name: &str) -> Result<Vec<GitHubMember>, Error> {
+        let mut members = Vec::new();
+        let mut page_num = 1;
+        loop {
+            let page: Vec<GitHubMember> = self
+                .prepare(
+                    true,
+                    Method::GET,
+                    &format!(
+                        "orgs/rust-lang/teams/{}/members?per_page=100&page={}",
+                        name, page_num
+                    ),
+                )?
+                .send()?
+                .error_for_status()?
+                .json()?;
+            let len = page.len();
+            members.extend(page);
+            if len < 100 {
+                break;
+            }
+            page_num += 1;
+        }
+        let invites: Vec<GitHubMember> = self
+            .prepare(
+                true,
+                Method::GET,
+                &format!("orgs/rust-lang/teams/{}/invitations?per_page=100", name),
```

Comment:
As before, I'd like to see validation on this that we're not running up against the limit.